### ISSUE TITLE
[RadListView] iOS changing orientation on a phone with notch causes rendering issues.

### ISSUE THREAD

**Issue by oslc15** [NativeScript/nativescript-ui-feedback#1347]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Tell us about the problem
changing the orientation of a view with a RadListView inside causes rendering issues. Seems that some of the rows does not expand to 100% width. This only happens on iphones with the notch, SafeAreaLayout.

### Which platform(s) does your issue occur on?
iOS, both emulator and devices with a screen notch (iPhone X, XS, XR, 11)

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI plugin version: "8.0.1"**
- **CLI: 6.1.2**
- **Cross-platform modules: 6.1.2**
- **Runtime(s): 6.1.1**

### Please tell us how to recreate the issue in as much detail as possible.
1. Start the application in an iPhone with a notch. iPhone X, XS, XR, 11
2. When the listview has loaded. try rotating the screen so the orientation becomes landscape.
3. Scroll through the list and see the bug appear on some om the lines.
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-05 at 12 16 50](https://user-images.githubusercontent.com/15787715/73837320-8159a080-4811-11ea-8395-d47e320a9a04.png)

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
[landscape.zip](https://github.com/NativeScript/nativescript-ui-feedback/files/4159168/landscape.zip)

**Comment by NickIliev** [comment]
@oslc15 try updating the tns-core-modules to latest version.

**Comment by oslc15** [comment]
@NickIliev Issue still remains after updating tns-core-modules. Have you got it working correctly in the demo with latest version Nick?

**Comment by mbokil** [comment]
I am also have this problem with "tns-core-modules": "^6.3.2" with RadPieChart. When I suspend the app and then resume in iOS iPad the color palette returns to default colors. Pre and post images and code I am using for DonutSeries.
![image](https://user-images.githubusercontent.com/1885868/74176795-6d91ae00-4c06-11ea-8a21-bf276a3ae8e8.png) 
![image](https://user-images.githubusercontent.com/1885868/74176902-9e71e300-4c06-11ea-9c15-2e23f4b7e565.png)

`<RadPieChart left="0" top="10"  height="180" width="180" allowAnimations="false" row="1">
  <DonutSeries v-tkPieSeries
  	seriesName="donut"
    selectionMode="DataPoint"
    expandRadius="0.4"
    outerRadiusFactor="1"
    innerRadiusFactor="0.8"
    valueProperty="count"
    legendLabel="status"
    :items="statsTable">
  </DonutSeries>

  <Palette v-tkPiePalette seriesName="donut">
    <PaletteEntry v-tkPiePaletteEntry fillColor="#78be21" strokeColor="#78be21"/>
    <PaletteEntry v-tkPiePaletteEntry fillColor="#ff8100" strokeColor="#ff8100"/>
    <PaletteEntry v-tkPiePaletteEntry fillColor="#9320A2" strokeColor="#9320A2"/>
		<PaletteEntry v-tkPiePaletteEntry fillColor="#d8d0d0" strokeColor="#d8d0d0"/>
	</Palette>
</RadPieChart>`

**Comment by oslc15** [comment]
@mbokil i am not sure your bug is the same as mine, i would guess that it should be put in a separate issue, as it is not the same component or way of reproducing as in mine.

**Comment by mbokil** [comment]
Yeah it looks like a problem with several components. List and RadChart. I see the issue has been logged with RadChart. [https://github.com/NativeScript/nativescript-ui-feedback/issues/1335]

**Comment by softwarebam** [comment]
me too if orientation change from portrait  to landscape
landscape issues on iphone 11

but landscape first load no problem

nativescript-ui-listview: "^8.0.1"
tns-core-modules: "^6.4.2"

**Comment by softwarebam** [comment]
<lv:RadListView  iosOverflowSafeAreaEnabled="false"

is Work for me
